views/appoint-director-submitted.html

```diff
@@ -17,9 +17,63 @@
       
       <form action="" method="post">
 
-        <h1 class="govuk-heading-l">Appointment submitted</h1>
-        <!-- This is the page for the appointment submitted screen -->
+        <div class="govuk-panel govuk-panel--confirmation">
+          <h1 class="govuk-panel__title">
+            Appointment submitted
+          </h1>
+          <div class="govuk-panel__body">
+            Reference number<br><strong>{{ referenceNumber }}</strong>
+          </div>
+        </div>
 
+        {{ govukSummaryList({
+          rows: [
+            {
+              key: {
+                text: "Company name"
+              },
+              value: {
+                text: companyName
```

Comment:
Do we need to make sure this is always uppercase?

Reply:
I believe company name is always stored in capitalised cases, also this implementation is identical to TM01 confirmation page.